src/less/components/breadcrumb/index.less

```diff
@@ -24,4 +24,4 @@
     font-size: 0.5rem;
     margin-right: 0.5rem;
   }
-}
+}
```

Comment:
Please revert the removal of the newline at the end of this file.

This change can be avoided by setting up [EditorConfig](https://editorconfig.org/) for your text editor, and using the `.editorconfig` file at the root of the project.